docker/hbase/Dockerfile

```diff
@@ -94,7 +94,7 @@ RUN mv /tmp/hadoop-etc/* /opt/hadoop/etc/hadoop
 RUN mv /tmp/hbase-etc/* /opt/hbase/conf/
 RUN mv /tmp/nutch-etc/* /opt/apache-nutch-2.x/conf/
 
-ENV JAVA_HOME /usr
+ENV JAVA_HOME /usr/lib/jvm/java-7-oracle
 ENV NUTCH_ROOT /opt/apache-nutch-2.x
```

Comment:
To avoid creating another layer could you chain the env variables creation as such?

```
ENV NUTCH_ROOT /opt/apache-nutch-2.x \
        JAVA_HOME /usr/lib/jvm/java-7-oracle
```
Thanks for the fix